main.css

```diff
@@ -0,0 +1,85 @@
+body {
+  background: #1c1a19;
+}
+
+.main-body {
+  background-image: url("pictures/Group.png");
+  background-position: 240px 270px;
+  background-repeat: no-repeat;
+  height: 570px;
+}
+
+header {
+  display: flex;
```

Comment:
- [ ] For DRY css you should combine duplicate properties. 
You can create class `.d-flex { display: flex}` and use this class in html file.

- For more information about DRY principles you can check this [article](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- Also you can refresh your knowledge about "html and css best practices" in [curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)